packages/compiler-dom/__tests__/transforms/vModel.spec.ts

```diff
@@ -63,6 +63,13 @@ describe('compiler: transform v-model', () => {
     expect(generate(root).code).toMatchSnapshot()
   })
 
+  test('input with v-bind shorthand type should use dynamic model', () => {
+    const root = transformWithModel('<input :type v-model="model" />')
```

Comment:
I'm wondering whether it might be better to write this test with the `:type` after the `v-model`? Or both?

I understand that the test is following the order from the other tests, but in practice the order `:type v-model="model"` already works when using the full compiler (including the `vBind` transform), it only fails when the `:type` comes after the `v-model`.